go/vt/sqlparser/parse_test.go

```diff
@@ -3489,6 +3489,44 @@ func TestLoadData(t *testing.T) {
 	}
 }
 
+func TestTrim(t *testing.T) {
+	testCases := []parseTest{
+		{
+			input:                `SELECT TRIM("foo")`,
+			output:               `select trim('foo', ' ', b) from dual`,
```

Comment:
So generally speaking, the Format function should produce output which is itself parseable, and this won't

Unless you have a good reason not to do it this way instead, you should change the Format function to return something similar to the input